.gitignore

```diff
@@ -0,0 +1,3 @@
+node-modules
```

Comment:
Kindly remove the misspelt file `node-modules` from your `.gitignore` file. This will make your code base clean and easy to understand.